### PR TITLE
Replace material UI components

### PR DESCRIPTION
**Pull request by Blubtastic**
# What does this PR change
1. removes components from the `@material-ui/` packages
2. replaces these with corresponding EDS components
3. rewrite logic to support EDS components
4. rewrite style to support EDS components

PR [#62](https://github.com/equinor/flow-diagram-explorer/pull/62) uninstalls the MUI packages

### CODE REVIEW

File: src/lib/components/Timeline/timeline.tsx [hunk 1]
```diff
@@ -252,59 +248,34 @@ export const Timeline: React.FC<TimelineProps> = (props: TimelineProps): JSX.Ele
...
-                            minDate={sortedTimeFrames.length > 0 ? sortedTimeFrames[0].startDate : undefined}
-                            className="FlowDiagramExplorer__Timeline__DatePicker"
-                            InputProps={{
-                                disableUnderline: true,
-                            }}
-                        />
-                    </MuiPickersUtilsProvider>
+                    <TextField
+                        id="timeline_date"
+                        type="date"
```
Comment (jsolaas) [inline src/lib/components/Timeline/timeline.tsx:267]: Does this work well? Did you test different browsers?

https://caniuse.com/input-datetime

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.0.0-alpha6] - 2021-10-05
 
+## Changed
+
+-   [#61](https://github.com/equinor/flow-diagram-explorer/pull/61) - Replaced all MUI components with EDS
+
```
Comment (jsolaas) [inline CHANGELOG.md:17]: This should probably be for next release?

File: src/lib/components/Timeline/timeline.tsx (hunk 1 above)
Comment (Blubtastic) [reply src/lib/components/Timeline/timeline.tsx:267]: yeah good point 😅 Seems to be working well on the major browsers. 

Setting the input type to `week` and `month` isn't supported in all browsers, but we're not using this. Min/Max isn't supported in some niche browsers, but that doesn't break anything it's just a worse user experience _(which you're probably used to if you're using those browsers)_

So i guess EDS could fix the missing browser support themselves, but I'm guessing it's not worth the time for them.

File: CHANGELOG.md (hunk 2 above)
Comment (Blubtastic) [reply CHANGELOG.md:17]: Yeah i don't know how changelogs work (and i have no idea how and when to release a new version 😅).

**Pull request merged**